src/index.js

```diff
@@ -2156,6 +2223,20 @@ class Compute extends common.Service {
   image(name) {
     return new Image(this, name);
   }
+  /**
+   * Get a reference to a Google Compute Engine instance template.
+   *
+   * @see [Instance Templates Overview]{@link https://cloud.google.com/compute/docs/instanceTemplates}
+   *
+   * @param {string} name - Name of the instance template.
+   * @returns {Image}
+   *
+   * @example
+   * var instanceTemplate = gce.instanceTemplate('instance-template-name');
+   */
+  instanceTemplate(name) {
```

Comment:
This method needs tests.

Reply:
Does it? I'm mostly interested in testing `new InstanceTemplate` which is tested in test/instance-template.js

Reply:
Yes, we test every method, so we would need a new describe block in test/index.ts that checks:

- should throw if an instance template name is not provided
  - catches the error and validates the contents of the message are as written in the source
- should properly create and return an InstanceTemplate issue
  - confirms that the arguments are correct
  - mocks "InstanceTemplate" and confirms what is returned in the mock is returned to the caller